cluster/ci/config/prow/openshift/config-updater/config-ci.yaml

```diff
@@ -11,6 +11,10 @@ data:
     - projects/prometheus/prometheus.yaml
     - projects/prometheus/node-exporter.yaml
     - projects/kubernetes/autoscaler.yaml
+    - projects/kubernetes/cluster-capacity.yaml
+    - projects/kubernetes/descheduler.yaml
+    - projects/kubernetes/metrics-server.yaml
+    - projects/kubernetes/node-problem-detector.yaml
```

Comment:
Why do we need to do it?

Reply:
So that when there is a PR merged for one of this files, a reapply will automatically trigger on api.ci.  Versus having to manually log in to api.ci and do a `oc apply` in addition to making the persistent change in this repo.